pyvista/core/filters/data_set.py

```diff
@@ -4340,6 +4340,42 @@ def shrink(self, shrink_factor=1.0, progress_bar=False):
             return output.extract_surface()
         return output
 
+    def tessellate(self, progress_bar=False):
+        """Tessellate a mesh.
+
+        This filter tessellate a mesh.
```

Comment:
From VTK's documentation:

```suggestion
        This filter approximates nonlinear FEM-like elements with linear
        simplices. The output mesh will have geometry and any fields specified
        as attributes in the input mesh's point data. The attribute's copy
        flags are honored, except for normals.
        
        For more details see `vtkTessellatorFilter <https://vtk.org/doc/nightly/html/classvtkTessellatorFilter.html#details>`_.
```

Reply:
What does

> The attribute's copy flags are honored, except for normals.

mean?

Reply:
Thanks @MatthewFlamm. If it isn't exposed in PyVista then it might be clearer not to include that in the docstring.